graphql/resolvers.go

```diff
@@ -2204,6 +2204,24 @@ func (r *queryResolver) InstanceTypes(ctx context.Context) ([]string, error) {
 
 type taskResolver struct{ *Resolver }
 
+func (r *taskResolver) DisplayTask(ctx context.Context, obj *restModel.APITask) (*restModel.APITask, error) {
+	t, err := r.sc.FindTaskById(*obj.Id)
+	if err != nil {
+		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("Could not find task with id: %s", *obj.Id))
+	}
+	dt, err := t.GetDisplayTask()
+	if dt == nil || err != nil {
+		return nil, nil
+	}
+	apiTask := &restModel.APITask{}
+	if err = apiTask.BuildFromService(dt); err != nil {
+	  return nil, InternalServerError.Send(ctx, fmt.Sprintf("Unable to convert display task: %s to APITask", dt.Id))
```

Comment:
the linter is complaining about whitespace here (I think this is supposed to be indented with a tab, but you have 2 spaces). Can you make sure `make lint-graphql` passes in your local branch?